### ISSUE TITLE
BIOS seems to be missing?

### ISSUE THREAD

**Issue by AlbeyAmakiir** [vircon32/vircon32-libretro#12]
The [libretro docs for Vircon32](https://docs.libretro.com/library/vircon32/) state:
> The Vircon32 core already contains the Vircon32 Standard BIOS. But you can optionally use a different BIOS file by placing it in the frontend's system directory, with name Vircon32Bios.v32.

But retroarch crashes when I run the core. Checking `Manage Cores`, the info for Vircon32 says
> (!) Missing, Required: StandardBios.v32 (Vircon32 BIOS 1.1)

**Comment by AlbeyAmakiir** [comment]
Actually, I'm not sure the crash is related to the BIOS. Giving it a bios from the site removes the "missing" message, but it still crashes immediately on loading the core, with no logs.

In case any of this is important, I'm on ArchLinux, X (not Wayland), NVidia GPU, Intel CPU. I tested the version on AUR and it works fine.

**Comment by vircon32** [comment]
Hi Albey. I will need you to check a couple of things. First ensure that the core that you have is updated: earlier versions did not include the embedded bios. Second, do you have an updated core info file for Vircon32? In the current info file the BIOS file is no longer stated as required.

Abour the crash, unfortunately I don't know much abour arch linux or AUR, so let's see if this fixes your crash...

**Comment by AlbeyAmakiir** [comment]
I should have the latest since I only installed it yesterday. After updating, the core says it's on version 2023.08.24. That doesn't sound like it should be correct. Might it be getting confused since `2023` is a larger number than `24`?

Info file:
```
# Software Information
display_name = "Vircon32"
authors = "Carra"
supported_extensions = "v32|V32"
corename = "Vircon32"
license = "3-clause BSD"
permissions = ""
display_version = "2023.08.24"
categories = "Game engine"

# Hardware Information
systemname = "Vircon32"
systemid = "vircon32"

# Libretro Features
supports_no_game = "true"
savestate = "false"
cheats = "false"
input_descriptors = "false"
memory_descriptors = "false"
libretro_saves = "false"
core_options = "false"
core_options_version = "1.0"
load_subsystem = "false"
hw_render = "true"
required_hw_api = "OpenGL >= 3.0 | OpenGL ES >= 2.0"
needs_fullpath = "true"
disk_control = "false"
is_experimental = "false"
database = "Vircon32"

# BIOS / Firmware
firmware_count = 1
firmware0_desc = "StandardBios.v32 (Vircon32 BIOS 1.1)"
firmware0_path = "StandardBios.v32"
firmware0_opt = "false"
notes = "(!) StandardBios.v32 (md5): 1ad96946bce7bd0422b7d9340735b248 | (!) Check https://github.com/vircon32/ConsoleSoftware/releases/tag/bios-v1.1 | (!) Homepage: http://www.vircon32.com/"

description = "Vircon32 is a virtual game console, inspired by classic 16 & 32 bit systems as well as the arcade era."
```

**Comment by vircon32** [comment]
I see, That info file is outdated. I thought this was updated on libretro after my latest changes, but for some reason it's not. I'll look into it. For now you can try if changing these settings manually in your info file fixes your problem:

firmware0_desc = "Vircon32Bios.v32 (alternative BIOS file)"
firmware0_path = "Vircon32Bios.v32"
firmware0_opt = "true"
notes = "Optional alternative BIOS file|Vircon32 Standard BIOS is embedded in the core"

**Comment by vircon32** [comment]
Hi again Albey. I think I was able to find the issue. I uploaded a new commit with the fix. The libretro buildbot should make the new version available tomorrow. To prevent mistakes I have also updated the core version to 2024.04.14. I'm also working to get the core info file updated. My previous update did happen, but someone accidentally reverted the file in another commit.

I hope this fixes your issues.
